packages/flutter_tools/lib/src/test/test_compiler.dart

```diff
@@ -91,12 +104,23 @@ class TestCompiler {
   /// Create the resident compiler used to compile the test.
   @visibleForTesting
   Future<ResidentCompiler> createCompiler() async {
+    // Copy lock-guarded testfile (if it exists) into this copy for this process only.
+    ensureDirectoryExists(testLockFilePath);
```

Comment:
Looking at the coverage diff it appears as if there is no coverage for this line

Reply:
Sure --- the createCompiler is overwritten in the test, so none of this is touch via the test.
I can change that so it runs super.createCompiler too...